admin_ui/app/permission/PermissionListFilters.js

```diff
@@ -122,7 +122,7 @@ const PermissionListFilters = () => {
             defaultChecked={filters.isSystemPermission}
             onChange={onSystemPermissionChage}
           />
-          Show system permissions
+          Show only system permissions
```

Comment:
How about turning this around, i.e. "Hide user permissions", as you suggested in our online call? I think it might be better for the user.